src/targets/node/nodeTarget.ts

```diff
@@ -57,7 +58,8 @@ export class NodeTarget implements Target {
     if (targetInfo.title)
       this._targetName = `${basename(targetInfo.title)} [${targetInfo.targetId}]`;
     else this._targetName = `[${targetInfo.targetId}]`;
-
+    if (skipFiles.length)
```

Comment:
Seems skipping is only enabled if I have configured it at the beginning, but how does that work with toggling?

Reply:
Good catch! Thank you! I've since fixed it.